tap_aircall/tap.py

```diff
@@ -8,17 +8,36 @@
 from tap_aircall.streams import (
     CallsStream,
     UsersStream,
+    UserStream,
+    TeamsStream,
+    TeamStream,
+    CallStream,
+    NumbersStream,
+    NumberStream,
+    ContactsStream,
+    ContactStream,
+    TagsStream,
+    TagStream
 )
-from tap_aircall._tap import _Tap
 # TODO: Compile a list of custom stream types here
 #       OR rewrite discover_streams() below with your custom logic.
 STREAM_TYPES = [
     CallsStream,
+    CallStream,
     UsersStream,
+    UserStream,
+    TeamsStream,
+    TeamStream,
+    NumbersStream,
+    NumberStream,
+    ContactsStream,
+    ContactStream,
+    TagsStream,
+    TagStream
 ]
 
 
-class Tapaircall(_Tap):
+class Tapaircall(Tap):
```

Comment:
Could you delete  following properties `auth_token`, `project_ids `& `api_url ` in Tap properties and README?

Reply:
Done